### PR TITLE
Updated how to determine if a node is ready or not

### PR DESCRIPTION
**Pull request by stevesloka**
Related to #37 this updates how the check for NodeStatus is done. Please take a look @sttts 

//cc @preillyme

### CODE REVIEW

**Comment by ohohoreilly** [comment]
LGTM but will wait for a :+1: from @sttts to merge

File: master/components/dashboard/js/modules/controllers/cAdvisorController.js [hunk 1]
```diff
@@ -22,7 +22,11 @@ app.controller('cAdvisorController', [
         $scope.minions = res;
 
         var promises = lodash.map(res.items, function(m) {
-            return cAdvisorService.getDataForMinion(m.metadata.name, (m.status.conditions[0].status === 'True'));
+            var readyCondition = m.status.conditions.filter(function ( obj ) {
+                return obj.type === 'Ready';
+            })[0];
```
Comment (sttts) [inline master/components/dashboard/js/modules/controllers/cAdvisorController.js:27]: There is no guarantee that the Ready condition exists at all.

Comment (ohohoreilly) [inline master/components/dashboard/js/modules/controllers/cAdvisorController.js:27]: What do you think would be the best way to perform this check @sttts?

Comment (stevesloka) [inline master/components/dashboard/js/modules/controllers/cAdvisorController.js:27]: I can check for that condition and default to no if it's not found, wasn't sure if that status was always returned or not.

Comment (sttts) [inline master/components/dashboard/js/modules/controllers/cAdvisorController.js:27]: `...[0]` will give undefined if there is no such condition. Just check for that case below: `readyCondition && readyCondition.status === 'True'`.

Comment (stevesloka) [inline master/components/dashboard/js/modules/controllers/cAdvisorController.js:27]: I like that better. I'll update again.

Comment (stevesloka) [inline master/components/dashboard/js/modules/controllers/cAdvisorController.js:27]: Ok pushed again and rebased.

Comment (sttts) [inline master/components/dashboard/js/modules/controllers/cAdvisorController.js:27]: Let me test against my cluster.

Comment (sttts) [inline master/components/dashboard/js/modules/controllers/cAdvisorController.js:27]: :+1:  works fine. Thanks a lot!

**Comment by sttts** [comment]
lgtm

**Comment by ohohoreilly** [comment]
LGTM

**Comment by sttts** [comment]
Can you build and push a v4 version with this patch?

**Comment by ohohoreilly** [comment]
@mikedanese can you please do the push for @sttts?

**Comment by jdef** [comment]
looks like v4 was pushed https://github.com/kubernetes/kubernetes/commit/b660cf9596149b23df4dd3df1f2d1e0bf9f5f2ce

**Pull request merged**